includes/functions.php

```diff
@@ -2080,86 +2074,88 @@ function tsml_import_has_changes($data_source_url, $meetings, $data_source_count
 		}
 	}
 
-	// create array of db slugs for matching
+	// create array of database slugs for matching
 	$db_slugs = array_column($db_meetings, 'slug', 'id');
 	sort($db_slugs);
-	$db_slugs_cnt = count($db_slugs);
 
-	// list changed and new meetings found in the ds feed
-	foreach ($meetings as $meeting) {
+	// list changed and new meetings found in the data source feed
+	foreach ($feed_meetings as $meeting) {
+	
+		$dayofweek = $dow_number = '';
+		tsml_get_day_of_week_info( $meeting['day'], $dayofweek, $dow_number);
```

Comment:
passing return values by reference is weird - how about

```
list($dayofweek, $dow_number) = tsml_get_day_of_week_info($meeting['day']);
```

Reply:
Using list was my first attempt which didn't work, so I tried option 2 of the 3 methods I found for returning multiple values from a function call. It seemed to work well. Is that what putting an ampersand in front of the variable does, make it by reference instead of by value? Hmm...